lib/index.js

```diff
@@ -100,35 +114,24 @@ function setupRateLimitEventExt(server, options) {
   };
 
   server.ext(event, (request, reply) => {
+    // This handler is going to be called one time per registration of patova
     // If current response is already not conformant we can stop
-    const currentRlRequest = store.get(request.id);
-    if (currentRlRequest && !currentRlRequest.isConformant()) { return; }
+    const currentRequestLimit = request.plugins.patova && request.plugins.patova.limit
+    if (currentRequestLimit && !currentRequestLimit.conformant) { return; }
```

Comment:
I think you can also check here if `request.response.isBoom`, I saw that somewhere else